### PR TITLE
chore(release): pull release/v1.62.0 into main

### PR DESCRIPTION
**Pull request by devops-github-rudderstack**
:crown: *An automated PR*

### CODE REVIEW

**Comment by devops-github-rudderstack** [comment]
Test report for this run is available at: https://test-integrations-dev.s3.amazonaws.com/integrations-test-reports/rudder-transformer/3276/test-report.html

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,43 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [1.62.0](https://github.com/rudderlabs/rudder-transformer/compare/v1.60.0...v1.62.0) (2024-04-15)
+
+
```
Comment (krishna2020) [inline CHANGELOG.md:7]: Duplicate change log again - @sanpj2292 can you check any action updates as part of CI/CD project leading to this ?

File: src/cdk/v2/destinations/reddit/utils.js [hunk 2]
```diff
@@ -28,6 +28,59 @@ const batchEvents = (successfulEvents) => {
...
+    }
+    // Proceed with calculation if not all prices are undefined
+    return properties.products.reduce(
+      (acc, product) => acc + (product.price || 0) * (product.quantity || 1),
+      0,
+    );
+  }
+  // For single product scenario, check if price is undefined
+  if (properties.price === undefined) {
+    return null; // Return null if price is undefined
```
Comment (krishna2020) [inline src/cdk/v2/destinations/reddit/utils.js:47]: @shrouti1507 Make use of codecov annotations to get 100% coverage for utils

File: src/controllers/delivery.ts [hunk 3]
```diff
@@ -1,28 +1,30 @@
...
 import { FixMe } from '../util/types';
+import tags from '../v0/util/tags';
+import { ControllerUtility } from './util';
 
 const NON_DETERMINABLE = 'Non-determinable';
 
 export class DeliveryController {
   public static async deliverToDestination(ctx: Context) {
-    logger.debug('Native(Delivery):: Request to transformer::', JSON.stringify(ctx.request.body));
+    logger.info('Native(Delivery):: Request to transformer::', ctx.request.body);
```
Comment (krishna2020) [inline src/controllers/delivery.ts:27]: Won't the log file increase by huge margin with every event getting logged ?

Comment (krishna2020) [reply src/controllers/delivery.ts:27]: need to be logger.debug itself

File: src/cdk/v2/destinations/reddit/utils.js (hunk 2 above)
Comment (shrouti1507) [reply src/cdk/v2/destinations/reddit/utils.js:47]: src/cdk/v2/destinations/reddit/utils.test.js file contains all of the edge cases. As discussed we will add component test case for each of them next time.

File: src/controllers/delivery.ts (hunk 3 above)
Comment (aashishmalik) [reply src/controllers/delivery.ts:27]: yes, will revert it

File: CHANGELOG.md (hunk 1 above)
Comment (krishna2020) [reply CHANGELOG.md:7]: yeah but why that's happening ? It's the same case from last 3 releases - why just the previous version is not being considered ?

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,30 @@
...
+* rakuten: adding a default value for tr ([#3240](https://github.com/rudderlabs/rudder-transformer/issues/3240)) ([3748f24](https://github.com/rudderlabs/rudder-transformer/commit/3748f24e21634fc74c5e5b3761551c64c8e69942))
+
+
+### Bug Fixes
+
+* adding check for reserved key words in extract custom fields  ([#3264](https://github.com/rudderlabs/rudder-transformer/issues/3264)) ([3399c47](https://github.com/rudderlabs/rudder-transformer/commit/3399c47fdce1b3d19e29306ca3c5692a2fbc30fb))
+* deployment file paths ([#3216](https://github.com/rudderlabs/rudder-transformer/issues/3216)) ([808727d](https://github.com/rudderlabs/rudder-transformer/commit/808727de17e400ed102a843ab3b30f81f8900f24))
+* email mappings ([#3247](https://github.com/rudderlabs/rudder-transformer/issues/3247)) ([791cbf5](https://github.com/rudderlabs/rudder-transformer/commit/791cbf55fc6940af4e3208212b82c891c6618fc3))
+* fixed userId mapping, now mapping to uid instead of id ([#3192](https://github.com/rudderlabs/rudder-transformer/issues/3192)) ([70a468b](https://github.com/rudderlabs/rudder-transformer/commit/70a468bf16ecd5ee0b6fecee4b837895d19c525f))
```
Comment (krishna2020) [inline CHANGELOG.md:21]: @sandeepdsvs can you fix with the right one ?

Comment (sandeepdsvs) [reply CHANGELOG.md]: Done

**Review by krishna2020 — DISMISSED** [review]

**Review by krishna2020 — APPROVED** [review]

**Review by sanpj2292 — APPROVED** [review]
